package.json

```diff
@@ -7,6 +7,7 @@
   "private": true,
   "dependencies": {
     "@types/enzyme": "3.10.3",
+    "@types/highlight.js": "^9.12.3",
```

Comment:
We normally use fixed versions in our dependencies, do you mind to remove the `^` here and, by the way, in the next line?

Reply:
Of course! 
(But the way, I'm SO glad you're online and already giving feedback! Sorry for the time it took me to get back to this. Also, this has been way more complicated than I anticipated, so I appreciate your patience)